### ISSUE TITLE
Updating to v2.4.8.x Causes "Houston" Warning

### ISSUE THREAD

**Issue by bdleitner** [TweakScale/TweakScale#333]
This morning I ran CKAN to update mods before starting the game and TweakScale updated to v2.4.8.2. When I started the game I got "Houston, we have a problem" warning claiming that I didn't have a module manager installed, even though I do.
[My Mod List after downgrading](https://github.com/TweakScale/TweakScale/files/15444315/20250525-BDL-ModList.txt)

Downgrading to v2.4.8.0 did not solve the issue, but downgrading back to the prior version v2.4.7.6 did.

## Intended outcome:
Updated Tweakscale should recognize my installed Module Manager v4.2.3

## Actual outcome:
"Houston, we have a problem" warning claiming no module manager was installed.

## How to reproduce the issue:
Upgrade to v.2.4.8.x and start the game with [this mod list](https://github.com/TweakScale/TweakScale/files/15444315/20250525-BDL-ModList.txt)

## KSP.log and additional data

I don't think that's relevant here since I aborted before the game finished loading.

**Comment by Lisias** [comment]
Yes, I just uploaded a fix for that. My apologies - I didn't properly tested exactly the use case the thing intended to replace. 😞 

Please update to [2.4.8.3](https://github.com/TweakScale/TweakScale/releases/tag/RELEASE%2F2.4.8.3). I fixed it on it.
